.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -11,14 +11,14 @@ assignees: ''
 A clear and concise description of what the bug is.
 
 **FreeBSD version**
-Paste the output of `uname -a`
+Paste the output of `uname -aKU`
```

Comment:
This should help to identify cases where an end user has omitted to update the user environment (`U`) as part of an OS update/upgrade routine.